src/vaishnavi/A23_ArrayExample2.java

```diff
@@ -0,0 +1,27 @@
+package vaishnavi;
+
+public class A23_ArrayExample2 {
+	int findSumOfNumFromString(String str) {
+		int sum = 0;
+		String temp = "";
+		for (int index = 0; index < str.length(); index++) {
+			char ch = str.charAt(index);
+			if (Character.isDigit(ch))
+				temp = temp + ch;
+			if ((Character.isLetter(ch) && !temp.isEmpty()) || index == str.length() - 1) {
```

Comment:
https://github.com/KrishnaKTechnocredits/JAVATechnoDec2021/pull/907/files
just take a look of this code, you will get idea for this.